Gateway/Request/RefundDataBuilder.php

```diff
@@ -241,6 +241,28 @@ protected function getOpenInvoiceData($payment)
             );
         }
 
+        // Adjustment positive
+        if ($creditMemo->getAdjustment() != 0) {
+            $description = "Adjustment";
+            if ($creditMemo->getAdjustmentPositive() > 0 && $creditMemo->getAdjustmentNegative() > 0) {
+                $description .= " - Positive | Negative";
+            } elseif ($creditMemo->getAdjustmentPositive() > 0) {
+                $description .= " - Positve";
+            } elseif ($creditMemo->getAdjustmentNegative() > 0) {
+                $description .=  " - Negative";
+            }
+
+            ++$count;
+            $formFields = $this->adyenHelper->createOpenInvoiceLineAdjustment(
+                $formFields,
+                $count,
+                $description,
+                $creditMemo->getAdjustment(),
```

Comment:
Will this amount take the Charged Currency config in consideration?

Reply:
Same problem as #1138 (#1197) for all refund based actions